spatial-tracking-explainer.md

```diff
@@ -197,37 +197,58 @@ function onSessionStarted(session) {
+### Poses
+On a frame-by-frame basis, developers can query the location of any `XRSpace` within another `XRSpace` via the `XRFrame.getPose()` function. This function takes the `space` parameter which is the `XRSpace` to locate and the `relativeTo` parameter which defines the coordinate system in which the resulting `XRPose` should be returned. The `transform` attribute of `XRPose` is an `XRRigidTransform` representing the location of `space` within `relativeTo`. 
+
+While the `relativeTo` parameter is an `XRSpace`, developers will often choose to supply a `XRReferenceSpace` as the `relativeTo` parameter so that coordinates will be consistent with those used for rendering. For more information on rendering, see the main [WebXR explainer](explainer.md).
 
 ```js
-  let pose = frame.getPose(xrSpace, xrReferenceSpace);
+  let pose = xrFrame.getPose(xrSpace, xrReferenceSpace);
   if (pose) {
     // Do a thing
   }
 ```
 
-The `emulatedPosition` attribute of `XRPose` indicates that the translation components of retrieved pose matrices may not be accurate. There are a number of reasons this might be the case. For example, a headset with orientation-only tracking capability may include position data to represent neck modeling. Another reason might be the underlying platform's tracking-loss behavior causes orientation data to be updated while it is unable to update position data. In these situations, the `emulatedPosition` attribute will be set to `true`.
+#### Tracking loss
+Developers should check initially that the result from `getPose()` is not null, as the pose of `space` within `relativeTo` may not have been established yet. For example, a viewer may not yet have been tracked within the application's `XRReferenceSpace`, or a motion controller may not yet have been observed after the user turned it on.
 
-#### Rays
-An `XRRay` object includes both an `origin` and `direction`, both given as `DOMPointReadOnly`s. The `origin` represents a 3D coordinate in space with a `w` component that must be 1, and the `direction` represents a normalized 3D directional vector with a `w` component that must be 0. The `XRRay` also defines a `matrix` which represents the transform from a ray originating at `[0, 0, 0]` and extending down the negative Z axis to the ray described by the `XRRay`'s `origin` and `direction`. This is useful for positioning graphical representations of the ray.
+However, once a pose is initially established for a viewer or input source, pose matrices should continue to be provided even during tracking loss. The `emulatedPosition` attribute of `XRPose` indicates that the position component of the retrieved pose matrix does not represent an actively tracked position. There are a number of reasons this might be the case. For example:
+* A viewer with orientation-only tracking, whose position within an `eye-level` reference space represents neck modeling.
```

Comment:
It seems compatible to me - the concept of [viewer](https://immersive-web.github.io/webxr/#viewer) remains a first-class concept in the spec.

Reply:
Why `viewer` and not `reference space`?

Reply:
In general, a single space "having tracking" is a squishy shorthand for the well-defined notion of whether _two_ particular spaces can be tracked relative to one another.  I'm using that here in the explainer to get folks aligned with some digestible examples before we get specific in the following paragraphs.

When using that shorthand, though, I tend to find myself thinking of a viewer (e.g. my headset) as "having tracking", rather than thinking of my primary reference space itself (e.g. my floor) as "having tracking".  While a viewer is a kind of reference space, being that generic here in this list of concrete examples felt like it would take away from readers grokking the overall concept.